sidewalk_params:
  my_param: 1000
  spin_freq: 6
  image_topic: /usb_cam/image_raw #/kinect2/qhd/image_color#/usb_cam/image_raw#
  depth_image_topic: /kinect2/qhd/image_depth_rect

  edge_side_offset: 50
  edge_marker_width: 6
  edge_start_offset: 35
  edge_side_offset_promile: 10
  detect_percent_of_image: 60

  undistort_image: true
  compute_edge_cloud: true

  display:
    orig: true
    result: true
    edge:
      raw: true
      valid: true
      fix: true
    points:
      old: true
      new: true

  glitchFrame:
    areaBufferSize: 6
    maxAreaDifference: 100
    webSize: 30
    max_slope_deviation: 30
    max_slope_variance: 70
    slope_perpendicular_spread: 2

  edge_points_distribution:
    edge_points_dist: 20

camera_capture_params:
  frame_rate: 10
  publish_topic_name: camera_image

  #define EDGE_MARKER_VERTICAL_POINT_DIST 50

  #define EDGE_START_OFFSET 30
  #define EDGE_END 500
